bridge/setu/util/common.go

```diff
@@ -173,8 +176,12 @@ func CalculateTaskDelay(cliCtx cliContext.CLIContext) (bool, time.Duration) {
 		}
 	}
 
+	// Change calculation later as per the discussion
```

Comment:
Can you explain briefly the logic used for delay?

Reply:
Currently it will multiply delay for every 1000 unconfirmed txns in mempool
For example if the current default delay is 12 Seconds
Then for upto 1000 txns it will stay as 12 only
For 1000-2000 It will be 24 seconds
For 2000-3000 it will be 36 seconds

Basically for every 1000 txns it will increase the factor by 1.

Reply:
Can you also add it as comment in the code?

Reply:
Okay

Reply:
Done